purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -854,7 +856,7 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
     }
 
     /**
-     * Invalidates the cache for purchaser information.
+     * Invalidates the cache for customer information.
      *
      * Most apps will not need to use this method; invalidating the cache can leave your app in an invalid state.
      * Refer to https://docs.revenuecat.com/docs/purchaserinfo#section-get-user-information for more information on
```

Comment:
we'll wanna update this link at some point, maybe we should make it a rev.cat link

Reply:
you're right! Will make a ticket

Reply:
https://app.shortcut.com/revenuecat/story/11643/update-invalidatecustomerinfo-link-in-sdks